plugins/lando-recipes/recipes/laravel/laravel.js

```diff
@@ -45,10 +45,14 @@ module.exports = function(lando) {
     config.php = _.get(config, 'php', '7.1');
 
     // Start by cheating
-    var build = lando.recipes.build(name, base, config);
+    const build = lando.recipes.build(name, base, config);
+
+    // Set the pantheon environment
+    const envPath = 'services.appserver.overrides.services.environment';
+    _.set(build, envPath, {APP_LOG: 'errorlog'});
```

Comment:
i think doing this might be an issue @dustinleblanc, eg we are replacing the entire environment (which might already contain upstream envvars). Testing to verify whether that is true right now.

Reply:
@dustinleblanc stand down! all clear here!